OBAKit/Stops/StopPreferencesViewController.swift

```diff
@@ -28,7 +28,7 @@ class StopPreferencesViewController: FormViewController {
 
         super.init(style: .insetGrouped)
 
-        title = OBALoc("stop_preferences_controller.title", value: "Sort & Filter Routes", comment: "Title of the Edit Stop preferences controller")
+        title = OBALoc("stop_preferences_controller.title", value: "Filter Routes", comment: "Title of the Edit Stop preferences controller")
```

Comment:
@aaronbrethorst localization strings needs to be updated to reflect this change.

Reply:
Assuming I set it up correctly, Transifex monitors pushes to `master` and will automatically pull in new/updated strings.